### PR TITLE
Adding Metamask Snap box

### PR DESCRIPTION
**Pull request by kevinbluer**
## PR Description 

- Adding Metamask snap box
- Removing redundant sample.json

### CODE REVIEW

**Review by Tamaraa01x — APPROVED** [review]

**Review by emilyJLin95 — APPROVED** [review]
LGTM

File: src/data/boxes.json [hunk 1]
```diff
@@ -273,6 +273,14 @@
 			"arbitrum"
 		]
 	},
+	"snap-box": {
+		"description": "Metamask Snap box",
+		"stars": 4,
+		"tags": [
+			"metamask",
+			"snap"
```
Comment (Montoya) [inline src/data/boxes.json:281]: Can this be "snaps" instead? Snaps is the API

Comment (kevinbluer) [reply src/data/boxes.json]: For sure, done (2d3911eb)

**Review by Tamaraa01x — COMMENTED** [review]
Get

**Pull request merged**